setup.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(name='RecoUtils',
+      version='2019.02',
```

Comment:
version, author and maybe description could be taken programmatically from here: https://github.com/Microsoft/Recommenders/blob/master/reco_utils/__init__.py#L8

I think it would be something like:
```
from reco_utils import VERSION

setup(name='RecoUtils',
      version=VERSION,
     ...
)
same for the others

```

Reply:
Done!